boards/bluepill-stm32f030c8/doc.txt

```diff
@@ -0,0 +1,65 @@
+/**
+@defgroup    boards_bluepill-stm32f030c8 STM32F030C8 based Bluepill
+@ingroup     boards
+@brief       Support for the STM32F030C8 based Bluepill
+
+## Overview
+
+The STM32F030C8 based Bluepill is a very cheap breakout board for the STM32F030C8 MCU.
+
+## Hardware
+
+![STM32F030C8 based Bluepill](https://camo.githubusercontent.com/f33ec5f4068e1dcc4c549dbe3dc70d2ed784c19abf1c177f9640835180f88005/68747470733a2f2f696d616765732d6e612e73736c2d696d616765732d616d617a6f6e2e636f6d2f696d616765732f492f3631474765592532424c306a4c2e5f41435f534c313030305f2e6a7067)
+
+### MCU
+| MCU        | STM32F030C8T6         |
+|:---------- |:--------------------- |
+| Family     | ARM Cortex-M0         |
+| Vendor     | ST Microelectronics   |
+| RAM        | 8 kiB                 |
+| Flash      | 64 kiB                |
+| Frequency  | up to 48MHz           |
+| FPU        | no                    |
+| Timers     | 8 (2x watchdog, 1 SysTick, 5x 16-bit) |
+| ADCs       | 1x 12-bit             |
+| UARTs      | 2                     |
+| SPIs       | 2                     |
+| I2Cs       | 2                     |
+| RTC        | 1                     |
+| Vcc        | 2.0V - 3.6V           |
+| Datasheet  | [Datasheet](https://www.st.com/en/microcontrollers-microprocessors/stm32f030f4.html) |
+| Reference Manual | [Reference Manual](https://www.st.com/resource/en/datasheet/stm32f030f4.pdf) |
+| Programming Manual | [Programming Manual](http://www.st.com/resource/en/programming_manual/dm00051352.pdf) |
+
+## Flashing the device
+
+The STM32F030C8 based Bluepill board does not include a programmer.
+You have to connect a separate ST-Link programmer to the (SW)DIO, (SW)CLK, GND and
+NRST pins on the board.
```

Comment:
This contradicts the PR description. Does it have an NRST pin, or not?

Reply:
unfortunately not :(